libs/langchain-community/src/tools/reddit.ts

```diff
@@ -0,0 +1,123 @@
+import { getEnvironmentVariable } from "@langchain/core/utils/env"; //"../../../../langchain-core/src/utils/env.js";
```

Comment:
removed unused comment

```suggestion
import { getEnvironmentVariable } from "@langchain/core/utils/env";
```